README.md

```diff
@@ -1 +1,63 @@
-# Group_E_Simulation_Life_Cycle
\ No newline at end of file
+# Group E: A solution for simulation lifecycle management
+
+ - Françoise Blanc
+ - Román Cárdenas Rodríguez
+ - Ghida Dandan
+ - Aidan Fahlman
+ - Bruno St-Aubin
+
+## Problem Statement
+
+In many application domains, Modeling and Simulation (M&S) has shown to be useful to study real world systems and to support decision-making through models that abstract the systems under study. M&S requires fewer resources and involves less risk than studying the system itself. It is a way to evaluate new or upgraded systems without compromising limited resources, interrupting operations, compromising safety, etc. However, building accurate models that adequately represent real-world systems is a difficult and time-consuming task that requires extensive domain knowledge and a deep understanding of the simulation method used. Because of the significant effort involve in understanding a specific domain and building the required models, the tendency in the field is to build single-use simulators.
```

Comment:
Just some typos that I found.
In the first sentence of this paragraph: real-world (with a hyphen) 
In the last sentence: involved (instead of involve)